File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        

        i = 0

        while i < len(nums):
            if nums[i] in nums[:i]:
                nums.pop(i)
            else:
                i+=1
        return len(nums)
        # l=1

        # for r in range(1, len(nums)):
        #     if nums[r]!=nums[r-1]:
        #         nums[l]=nums[r]
        #         l+=1
        # return l
            

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.rs
impl Solution {
    pub fn find_max_consecutive_ones(nums: Vec<i32>) -> i32 {

        let mut res = usize::MIN;
        let mut cnt = 0;

        for i in 0..nums.len(){
            if nums[i] == 0 {
                
                cnt = 0;
            } else{
                cnt += 1;

            }
            res = res.max(cnt);
            

        }

        res as i32


    }
}
